earth_enterprise/src/installer/common.sh

```diff
@@ -80,6 +80,26 @@ xml_file_get_xpath()
         tail -n +2 | head -n -1
 }
 
+is_package_installed()
+{
+    local package_installed_retval=1
+
+    # args: $1: Ubuntu package
+    # args: $2: RHEL package
+
+    if [ "$MACHINE_OS" == "$UBUNTUKEY" ] && [ ! -z "$1" ]; then
+        if [[ ! -z "$(dpkg --get-selections | sed s:install:: | sed -e 's:\s::g' | grep ^$1)" ]]; then
```

Comment:
Minor: You could use a single `grep` pattern to tell if a package is installed, rather than piping twice through `sed`, and again through `grep`.  Perhaps, a `dpkg --status` output may be easier to parse, as well.

Reply:
I agree, that’ll simplify things. I’ll make the change when I get home this evening.